spec/controllers/api/visits_controller_spec.rb

```diff
@@ -15,6 +15,10 @@
     JSON.parse(response.body)
   }
 
+  before do
+    request.headers['Accept-Language'] = 'cy'
+  end
```

Comment:
It seemed like a good idea as it found missing translations, I didn't find it to be a problem and I don't know welsh...

Happy to create a subset of tests to cover localisation.